### PR TITLE
Add Archlinux Installation Documentation

### PR DESCRIPTION
**Pull request by 0pcom**
I have added PKGBUILDS for the pterodactyl panel and daemon to the archlinux user repos:
https://aur.archlinux.org/packages/pterodactyl-panel
https://aur.archlinux.org/packages/pterodactyl-daemon

These can be built into p installed with yay, which will automatically handle the dependancies.

Scripts and a systemd service which were detailed in the other documenation are included in the built package. These are noted in the installation guide.

### CODE REVIEW

**Comment by parkervcp** [comment]
You didn't update the `.vuepress/config.js ` file to add your guides;

**Comment by 0pcom** [comment]
I added it, sorry

**Comment by parkervcp** [comment]
Your daemon page still doesn't load.

**Comment by 0pcom** [comment]
Should be good now

**Comment by parkervcp** [comment]
We are now at my other comments about just linking them to the points in the main install docs for configuring the panel and daemon.

It is already covered. There is no reason to cover it twice.

**Comment by 0pcom** [comment]
The point of that was that I added a script at the package level which uses dialog that gives a menu of the configuration options. I found it easier to follow, to preface the daemon configuration section with that, however I can link per your specifications.

**Comment by parkervcp** [comment]
Considering this is for Arch I assume these users are more advanced and can figure a ton of this out on their own.

**Comment by 0pcom** [comment]
Should be good now

File: community/installation-guides/panel/archlinux.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+```
+
+The default Redis install is perfectly fine for the panel. If you have Redis already in use you may want to look into
+[running another Redis instance](https://community.pivotal.io/s/article/How-to-setup-and-run-multiple-Redis-server-instances-on-a-Linux-host).
+
+### Additional Utilities
+
+#### Certbot
+```bash
+apt install -y certbot certbot-nginx
```
Comment (andreas-fahrecker) [inline community/installation-guides/panel/archlinux.md]: Why use apt install in arch, especially if you have installed yay?

**Comment by arcdigital** [comment]
stale, closing

**Pull request closed without merging**